README.md

```diff
@@ -83,9 +85,8 @@ Building the documentation copies the full markdown into the `website` folder, r
 
 With the repository cloned to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-fastly`:
 
-* To explicitly build the documentation:  
-`go run scripts/website/parse-templates.go `  
+* To explicitly build the documentation:
+`go run scripts/website/parse-templates.go `
```

Comment:
Is there a way to *implicitly* build the documentation?

Reply:
Hmm that's a good question. Probably, but I would imagine that it would be a lot of work. We _may_ be better off to just _always_ generate the documentation.

@phamann is way more knowledgeable when it comes to the generated docs.

Reply:
If not just remove the word 'explicitly' :-)